rust/operator-binary/src/controller.rs

```diff
@@ -359,6 +392,7 @@ pub async fn reconcile_trino(trino: Arc<TrinoCluster>, ctx: Arc<Ctx>) -> Result<
                 &merged_config,
                 authentication_config.as_ref(),
                 &catalogs,
+                &rbac_sa.name_unchecked(),
```

Comment:
I dont like the `name_unchecked` since it can crash the operator. This will be removed with the build_rbac_resources PR from operator-rs. So nothing todo!